### ISSUE TITLE
readline: libreadline.so.8 not found (x86_64)

### ISSUE THREAD

**Issue by kenmays** [haikuports/haikuports#4414]
runtime_loader: libreadline.so.8 exists, but not found during recent compilations. 
```
chroot has these packages active:
readline7-7.0.5-1_x86_64.hpkg
```
NOTE: lib:libhistory >=7 pulls in readline7-7.0.5-1
[readline7.txt](https://github.com/haikuports/haikuports/files/3917050/readline7.txt)

Tested: hrev52295+132 x86_64, readline-8.0.1-1-x86_64.hpkg

**Comment by kenmays** [comment]
lib:libhistory >=7 pulls in readline7-7.0.5-1

**Comment by korli** [comment]
you probably need to rebuild bash and update with it.

**Comment by kenmays** [comment]
Rebuilt bash, bumped libhistory to ">=8"... haikuporter fixed.

**Comment by korli** [comment]
fixed with bash-5.0.011-3
